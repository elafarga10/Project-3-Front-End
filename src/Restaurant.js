import React from 'react';
import { Card } from 'react-bootstrap';
import { SideNav } from './Sidebar';

export const RestaurantPage = (props) => (
					<>
						<Card>
							<Card.Header>{props.restaurant[0].name}</Card.Header>
						</Card>{' '}
						<SideNav
							handleTitleChange={props.handleTitleChange}
							handleSummaryChange={props.handleSummaryChange}
							handleRevisitChange={props.handleRevisitChange}
							handlePostClick={props.handlePostClick}
							revisit={props.revisit}
							title={props.title}
							summary={props.summary}
						/>
					</>
				);
