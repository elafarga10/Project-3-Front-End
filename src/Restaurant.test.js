import React from 'react';
import { shallow } from 'enzyme';
import  Restaurant  from './Restaurant';

describe('Restaurant', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Restaurant debug />);
        expect(component).toMatchSnapshot();
    });
});