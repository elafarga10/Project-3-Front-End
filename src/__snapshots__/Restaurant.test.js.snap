// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restaurant should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
